frontend/apps/reader/modules/readeruserhyph.lua

```diff
@@ -29,7 +29,7 @@ end
 -- Unload is done automatically when a new dictionary is loaded.
 function ReaderUserHyph:loadDictionary(name, reload)
     if G_reader_settings:isTrue("hyph_user_dict") and lfs.attributes(name, "mode") == "file" then
-        local ret = self.ui.document:setUserHyphenationDict(name, reload)
+        local ret = cre.setUserHyphenationDict(name, reload)
```

Comment:
Might add before some logger.dbg() (you had them in credocument.lua, might be useful):
`logger.dbg("set user hyphenation dict", name, reload)`
and below
`logger.dbg("reset user hyphenation dict")`